server/events/metrics/common.go

```diff
@@ -10,6 +10,26 @@ const (
 	FilterAbsentMetric  = "absent"
 	FilterErrorMetric   = "error"
 
-	RootTag = "root"
-	RepoTag = "repo"
+	RootTag         = "root"
+	RepoTag         = "repo"
+	WorkflowTypeTag = "workflow_type"
+	ActivityTypeTag = "activity_type"
+	TaskQueueTag    = "task_queue"
+	NamespaceTag    = "namespace"
```

Comment:
I don't think this tag is used yet so I'm trying to understand what you mean by namespace here since you already wrap the concept of namespaces around the metrics handler itself. Is there a difference?

Reply:
oops this was initially meant to be the temporal namespace until i realized that metrics in temporal by default come with that tag and then i forgot to remove it. Will remove now.